rules-topics/15_smithing.md

```diff
@@ -0,0 +1,99 @@
+#### Key
+   >This recipe allows the Smith to create a key that is keyed for a particular lock by an ID code that both the key and the lock share. Keys will only work for the lock that they are keyed for. To make a Key for a lock, you must bring the lock to Logistics.
+#### Lock
+   >This recipe allows the Smith to create a lock able to keep out unskilled burglars. Lock physical representations are attached to the main door of a structure. This lock indicates that all portals are locked. The tag for the lock will be attached to the back of the lock presentation, and will note the quality of the lock as well as the lock ID code. To be able to enter a locked structure, a character must have a key tag that is ID associated with the lock. A character with a Key may unlock a Locked door on a 3 count, and may re-lock it on another 3 count. The rating of this lock is 1. This has 5 Durability.
+#### Rusty Weapon
+>The sign of a poor smith, or a genius, depending on who you ask. This process intentionally rusts the weapon to make it inflict foul wounds upon those it strikes. This process changes the base damage type of the weapon to Poison, at the cost of reducing it's Durability by 2. A Rusty Weapon cannot be Silvered.
+
+
+### Tier II
+
+#### Armor Resizing Kit
+> This kit has everything a blacksmith needs to adjust suits of armor to better suit their needs. A Smith using this kit may increase an existing suit of armor's value rating by spending additional resources. This does not change the durability of the armor. This has 5 charges.
+#### Lockpicks
+   >This is a basic item for use with the Lockpicking skill. Having lockpicks lowers the rating of the lock by 1. Lockpicks have 10 charges.
+#### Suit of Armor
+   >This creates a finely made suit of armor that has an Armor rating of 10, with 5 Durability. You can increase the Tier of the recipe and the cost in materials by 1 to increase the Armor Rating by 10, up to a max of 80.
+
+
+
+### Tier III
+
+#### Hardening
+   >This will harden a weapon, shield, armor or magic item to Resist Destruction once. An item may only be Hardened up to three times (For a maximum of Three Resist Destructions) May be applied to untagged items. (Such as Doors).
+#### Knight-Night Helmet
+   >This specially designed helmet for knights who have too often fell victim to physically-induced-narcolepsy. This helmet will let the wearer Minimize any Waylay effect they receive. This has 5 charges, and must be repped.
+#### Repair Lock
+   >The smith is able to fix a previously broken lock. The repair of the lock takes 1 minute of  time in addition to the three ranks of raw materials. This is not a tagged item.
```

Comment:
Extra space in `1 minute of time`.

`three ranks of raw materials` should be `three tiers of raw materials`.